fli-iam/shanoir-ng
# [fli-iam#1867] Restart monitoring of unfinished VIP jobs on startup

How to test :
- Launch a "big" VIP execution (one that will take several minutes)
- Check that an "Execution monitoring" job has been created
- Stop the Shanoir instance : `dk stop ofsep-qualif*`
- Restart the Shanoir instance : `dk start ofsep-qualif*`
- Reconnect to Shanoir and check that the corresponding job is visible and updated (note that it will probably in a fail state, due to Shanoir shutdown)
